modules/openapi-generator/src/main/resources/dart/dart-keywords.txt

```diff
@@ -62,6 +62,7 @@ throw
 true
 try
 typedef
+update
```

Comment:
was this added due to `built_value`?

Reply:
I don't understand this one, need some more context.

Reply:
@kuhnroyal because in the generated classes in `built_value` serializer, there is an update method

Reply:
Ok, for this you should use the `--reserved-words-mappings=update=foo` parameter, not add it to the language keywords.

Reply:
There really is an endless amount of possible collisions with different APIs, not sure.
But the approach from that commit is much better.